google/cloud/storage/bucket.py

```diff
@@ -469,9 +473,11 @@ def copy_blob(self, blob, destination_bucket, new_name=None,
         copy_result = client.connection.api_request(
             method='POST', path=api_path, _target_object=new_blob)
         new_blob._set_properties(copy_result)
+        if preserve_acl:
+            new_blob.acl.save(blob.acl)
         return new_blob
 
-    def rename_blob(self, blob, new_name, client=None):
+    def rename_blob(self, blob, new_name, client=None, preserve_acl=False):
```

Comment:
I agree, I believed that the ACL wasn't being preserved.

This has been removed.